packages/bp/src/admin/ui/src/workspace/bots/CreateBotModal.tsx

```diff
@@ -208,13 +221,58 @@ class CreateBotModal extends Component<Props, State> {
             {this.state.categories.length > 0 && (
               <FormGroup label={lang.tr('admin.workspace.bots.create.category')}>
                 <Select
-                  tabIndex="4"
                   options={this.state.categories}
                   value={this.state.selectedCategory}
                   onChange={selectedCategory => this.setState({ selectedCategory: selectedCategory as any })}
                 />
               </FormGroup>
             )}
+            <FormGroup
+              label={lang.tr('admin.workspace.bots.create.cloud')}
+              labelFor="checkbox-bot-cloud"
+              helperText={lang.tr('admin.workspace.bots.create.cloudHelper')}
+            >
+              <Checkbox
+                id="checkbox-bot-cloud"
+                label={lang.tr('admin.workspace.bots.create.cloudCheckbox')}
+                checked={this.state.isCloudBot}
+                onChange={e =>
+                  this.setState({
+                    isCloudBot: e.target.checked
+                  })
+                }
+              />
+            </FormGroup>
+            {this.state.isCloudBot && (
+              <FormGroup
+                label={lang.tr('admin.workspace.bots.create.cloudConfiguration')}
+                labelFor="cloud-client-id"
+                helperText={lang.tr('admin.workspace.bots.create.cloudConfigurationHelper')}
+              >
+                <InputGroup
+                  id="cloud-client-id"
+                  placeholder={lang.tr('admin.workspace.bots.create.clientIdPlaceholder')}
+                  value={this.state.cloudClientId}
+                  className={style.clientId}
+                  onChange={e =>
+                    this.setState({
+                      cloudClientId: e.target.value
+                    })
+                  }
+                />
+                <InputGroup
+                  id="cloud-client-secret"
+                  placeholder={lang.tr('admin.workspace.bots.create.clientSecretPlaceholder')}
+                  value={this.state.cloudClientSecret}
+                  type="password"
+                  onChange={e =>
+                    this.setState({
+                      cloudClientSecret: e.target.value
+                    })
+                  }
+                />
```

Comment:
I think you should add (basic) validation to the input groups like we do with the boitId and bot name inputs. Because currently, it is possible to check the `Create a Botpress cloud-ready bot` option and not enter any values.